server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -76,6 +76,18 @@ trait Endpoints extends BaseEndpoints {
       .errorOutVariantPrepend(conflictErrorOneOfVariant)
   }
 
+  protected val patchPartitioningParentEndpointV2
+  : PublicEndpoint[(Long, Long, String), ErrorResponse, SingleSuccessResponse[
+    ParentPatchV2DTO
+  ], Any] = {
+    apiV2.get
+      .in(V2Paths.Partitionings / path[Long]("partitioningId") / V2Paths.Parents / path[Long]("partitioningId") / V2Paths.Parents )
+      .in(query[String]("byUser"))
```

Comment:
the should be no query parameters, only request body ...

Reply:
Understood, will fix.